src/lib/FlightTasks/tasks/FlightTask/FlightTask.cpp

```diff
@@ -90,6 +90,11 @@ void FlightTask::_evaluateVehicleLocalPosition()
 	_yaw = NAN;
 	_dist_to_bottom = NAN;
 
+	if ((_time_stamp_current - _sub_attitude->get().timestamp) < _timeout) {
+		// yaw
+		_yaw = matrix::Eulerf(matrix::Quatf(_sub_attitude->get().q)).psi();
```

Comment:
Can you remove the yaw assignment further down? https://github.com/PX4/Firmware/pull/10803/files#diff-76c7cef38691d384b72e5f1f8695eb69R122

Reply:
Yes, I believe so, let's see if the tests  pass.

Reply:
It flies fine in SITL using ekf2.